tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/CallStackDebugTest.java

```diff
@@ -65,7 +65,7 @@ public void stackFrameDebugTest() throws BallerinaTestException {
         debugTestRunner.assertCallStack(frames[2], "func2", 23, "main.bal");
         debugTestRunner.assertCallStack(frames[3], "func1", 19, "main.bal");
         debugTestRunner.assertCallStack(frames[4], "addition", 14, "main.bal");
-        debugTestRunner.assertCallStack(frames[5], "start:f1", 2, "main.bal");
+        debugTestRunner.assertCallStack(frames[5], "start:anonymous", 2, "main.bal");
```

Comment:
Needs to fix this

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/30636 to track this